install_config/persistent_storage/persistent_storage_fibre_channel.adoc

```diff
@@ -51,34 +40,29 @@ spec:
+<1> Optional: World wide identifiers (WWIDs). Either FC `*wwids*` or a combination of FC `*targetWWNs*` and `*lun*` must be set, but not both simultaneously. The FC WWID identifier is recommended over the WWNs target because it is guaranteed to be unique for every storage device, and independent of the path that is used to access the device. The WWID identifier can be obtained by issuing a SCSI Inquiry to retrieve the Device Identification Vital Product Data (`page 0x83`) or Unit Serial Number (`page 0x80`). FC WWIDs are identified as `/dev/disk/by-id/` to reference the data on the disk, even if the path to the device changes and even when accessing the device from different systems.
+<2> Optional: World wide names (WWNs). Either FC `*wwids*` or a combination of FC `*targetWWNs*` and `*lun*` must be set, but not both simultaneously. The FC WWID identifier is recommended over the WWNs target because it is guaranteed to be unique for every storage device, and independent of the path that is used to access the device. FC WWNs are identified as `/dev/disk/by-path/pci-<IDENTIFIER>-fc-0x<WWN>-lun-<LUN#>`, but you do not need to provide any part of the path leading up to the `WWN`, including the `0x`, and anything after, including the `-` (hyphen).
 
 [IMPORTANT]
 ====
-Changing the value of the `*fstype*` parameter after the volume has been
-formatted and provisioned can result in data loss and pod failure.
+Changing the value of the `*fstype*` parameter after the volume has been formatted and provisioned can result in data loss and pod failure.
 ====
 
 [[enforcing-disk-quotas-fibre]]
 
 === Enforcing Disk Quotas
-Use LUN partitions to enforce disk quotas and size constraints. Each LUN is one persistent volume. Kubernetes enforces
-unique names for persistent volumes.
+Use LUN partitions to enforce disk quotas and size constraints. Each LUN is one persistent volume. Kubernetes enforces unique names for persistent volumes.
 
-Enforcing quotas in this way allows the end user to request persistent storage
-by a specific amount (e.g, 10Gi) and be matched with a corresponding volume of
-equal or greater capacity.
+Enforcing quotas in this way allows the end user to request persistent storage by a specific amount (e.g., 10Gi) and be matched with a corresponding volume of equal or greater capacity.
 
 [[volume-security-fibre]]
 
 === Fibre Channel Volume Security
-Users request storage with a `*PersistentVolumeClaim*`. This claim only lives in
-the user's namespace and can only be referenced by a pod within that same
-namespace. Any attempt to access a persistent volume across a namespace causes
-the pod to fail.
+Users request storage with a `*PersistentVolumeClaim*`. This claim only lives in the namespace of the user and can only be referenced by a pod within that same namespace. Any attempt to access a persistent volume across a namespace causes the pod to fail.
```

Comment:
access a pvc across a namespace cause the pod to fail.

Reply:
Resolved